jekyll/_cci2/vs-code-extension-overview.adoc

```diff
@@ -186,32 +186,101 @@ image::{{site.baseurl}}/assets/img/docs/vscode-ext-config-test-run-crop.png[Scre
+
+[#branch-protection]
+==== Branch protection
+This feature allows running arbitrary configs on any branch, including protected branches. Pipelines triggered this way will have access to the same **environment variables, contexts and OIDC tokens** as if they were triggered on the corresponding branch from VCS by the same user.
+
+link:https://circleci.com/docs/oidc-tokens-with-custom-claims/[OIDC tokens] make a number of cryptographically verifiable claims about a pipeline execution, including some claims about the VCS. As the unversioned config is not provided by a VCS, these claims should not be made on a pipeline with unversioned config. In the current state, an unversioned config could be used to exploit the claims in the OIDC token to affect production.
+
+This increases the risk of secret exfiltration, and could let users access cloud resources they should not be able to.
+
+[#vcs-status-checks]
+==== VCS Status checks
+Triggering pipelines with unversioned config may allow bypassing VCS status checks for the corresponding commits.
+
+This is a known issue that we are committed to resolve in the coming weeks.
+
+[#access-and-permissions]
+==== Access and permissions
+link:https://circleci.com/docs/contexts/#restrict-a-context-to-a-security-group-or-groups[Context restrictions based on groups or security groups] are respected. We recommend using groups to exclude context access to untrusted users.
+
+Per-user, per-project, per-branch and per-org permission checks are also respected by pipelines triggered with unversioned config.
+
+[#config-policies]
+==== Config Policies
+link:https://circleci.com/docs/config-policy-management-overview/[Config policy rules] apply to unversioned configs as they do to VCS-provided configs. This means that an unversioned config that does not pass a policy rule will trigger a policy fail, just like a VCS-provided config. However, if config policies differ per branch, it could be possible exploit these differences to run an unversioned config on a GitHub-protected branch, without going through a code merge.
+
+[#auditability]
+==== Auditability
+
+Audit logs trace when organization and project level settings are changed for this feature.
+
+- The event `organzation.settings.update` is sent when organization-level settings for this feature are changed.
+- The event `project.settings.update` is sent when project-level settings for this feature are changed.
+
+Both the events below log:
+
+- Date and time of the event happened
+
+- User ID of the actor performing the action
+
+- Information on which setting was changed
+
+The events above do not call out the new value explicitly, but this is inferable by looking at the current state of the setting.
+
+At this time, audit logs **do not** explicitly call out if a pipeline was run with unversioned config.
 
+[#coming-soon]
+==== Coming soon
```

Comment:
Can we make this a smaller header? The section below still belongs to "Auditability"